runelite-client/src/main/java/net/runelite/client/plugins/groundmarkers/GroundMarkerPlugin.java

```diff
@@ -65,6 +65,7 @@
 {
 	private static final String CONFIG_GROUP = "groundMarker";
 	private static final String MARK = "Mark tile";
+	private static final String CLEAR = "Unmark nearby tiles";
```

Comment:
I would probably rename this to Unmark visible tiles, nearby is a bit too cryptic.